README.md

```diff
@@ -996,7 +996,6 @@ Table of Contents
   * [txti.es](https://txti.es/) — Quickly create web pages with markdown.
   * [Vercel](https://vercel.com/) — Build, deploy, and host web apps with free SSL, global CDN, and unique Preview URLs each time you `git push`. Perfect for Next.js and other Static Site Generators.
   * [Versoly](https://versoly.com/) — SaaS focussed website builder - unlimited websites, 70+ blocks, 5 templates, custom CSS, favicon, SEO and forms. No custom domain.
-  * [Qovery](https://www.qovery.com) — Qovery is the simplest way to deploy your full-stack apps on AWS, GCP and Azure. It is free web hosting for developers with Database, SSL, a global CDN, and auto deploys from Git.
```

Comment:
They do still appear to provide a free tier?

Reply:
In the past, its free program provided server settings. Later, they changed it to a community program. They need to continue to contribute to the qovery community in order to get free points See:  https://hub.qovery.com/docs/community/reward/feedback/
The current free plan needs to provide its own infrastructure. I don't think it belongs to as-a-service offers

Reply:
Ah, I see once you click signup a bunch of b/s starts, yah lets remove them then.